ext/standard/config.m4

```diff
@@ -455,7 +455,7 @@ PHP_NEW_EXTENSION(standard, array.c base64.c basic_functions.c browscap.c crc32.
                             http_fopen_wrapper.c php_fopen_wrapper.c credits.c css.c \
                             var_unserializer.c ftok.c sha1.c user_filters.c uuencode.c \
                             filters.c proc_open.c streamsfuncs.c http.c password.c \
-                            random.c,,,
+                            random.c hrtime.c,,,
```

Comment:
Oh, i thought we'd rather exclude it in configure, but works this way, too. Except probably only a few can test it :/ There are no cases, i'd be aware of otherwise.